tests/foreman/smoke/test_ui_smoke.py

```diff
@@ -475,8 +472,8 @@ def test_puppet_install(self):
                 vm.register_contenthost(activation_key_name, org_name)
                 vm.configure_puppet(rhel6_repo)
                 host = vm.hostname
-                session.nav.go_to_hosts()
                 set_context(session, org=ANY_CONTEXT['org'])
+                session.nav.go_to_hosts()
```

Comment:
:+1: And this approach seems more valid - to select org/loc first and only then navigate through pages.
I wonder why selenium clicks on 'manage location' link... Mind filling an issue for that so we don't forget about it?